Formula/cryptominisat.rb

```diff
@@ -0,0 +1,37 @@
+class Cryptominisat < Formula
+  desc "Advanced SAT solver"
+  homepage "https://www.msoos.org/"
```

Comment:
Let's use the github page rather than the blog as homepage.

Reply:
How about using this: https://www.msoos.org/cryptominisat5/?